packages/theme-default/package.json

```diff
@@ -13,11 +13,15 @@
   "jsnext:source": "./src/index.ts",
   "types": "./dist/bundle.d.ts",
   "main": "./dist/index.js",
+  "imports": {
+    "#theme/*": "./src/*"
```

Comment:
aliases for self referencing. unlike `paths` in tsconfig.json, it's supported by modern resolvers and node.js.